lib/membrane_rtmp_plugin/rtmp/source/messages/on_meta_data.ex

```diff
@@ -0,0 +1,76 @@
+defmodule Membrane.RTMP.Messages.OnMetaData do
+  @moduledoc """
+  Defines the RTMP `onMetaData` command (sent by nginx client).
+  """
+
+  @behaviour Membrane.RTMP.Message
+
+  alias Membrane.RTMP.AMF.Encoder
+
+  defstruct ~w(
+    width height duration framerate video_data_rate video_codec_id audio_data_rate audio_codec_id
+  )a
+
+  @attributes_to_keys %{
```

Comment:
Nit: hardcoding keys-to-attributes results in a bit shorter notation. Also, you could `defstruct Map.keys(keys_to_attributes)`